core/src/test/resources/org/apache/calcite/test/SqlToRelConverterTest.xml

```diff
@@ -673,7 +707,7 @@ LogicalProject(DEPTNO=[$7], EXPR$1=[$9])
     </Resource>
     <Resource name="planNotExpanded">
       <![CDATA[
-LogicalProject(DEPTNO=[$7], EXPR$1=[$SCALAR_QUERY({
+LogicalProject(variablesSet=[[$cor0]], DEPTNO=[$7], EXPR$1=[$SCALAR_QUERY({
```

Comment:
`variablesSet` looks really odd here. What do you think of `correlation=[[$cor0]]`?

Reply:
A `correlation` sounds more natural to me too, but `variablesSet` is consistent with `LogicalFilter`. Do we need to change this too?

Reply:
ok, renamed this term for project only for now